modules/deploy/pages/redpanda/manual/resilience/shadowing-guide.adoc

```diff
@@ -0,0 +1,194 @@
+= Shadowing Guide
```

Comment:
@paulohtb6 Can we talk about where this Resilience section will go? Are you planning to keep it under Deploy and include Shadowing, HA, sizing, etc.? Maybe it makes sense for SM if there's separate info for K8s, but for Cloud I think we want it above this, like what we talked about [here](https://developer.hashicorp.com/consul/docs/concept/reliability).

Reply:
And I agree with Joyce that this page should be nested under shadowing.adoc. For title, I like **Configure Shadowing for Disaster Recovery**.

Reply:
@paulohtb6 Not sure why you are using "resilience" in this path--there is no resilience guide.
Let's keep the logic of the navtree in tact here. Currently, under both Cloud and Self-Managed > Depoly, we see possible platforms/products from which you can deploy. So placing Shadowing here under Linux breaks our org model. 
Please place the entry for Shadowing (or Disaster Recovery) under Manage:
```
Manage
 --> Shadowing
           --> Failover Quickstart (what you are now calling the Shadowing Guide)
```